cms/cms_toolbars.py

```diff
@@ -313,13 +313,16 @@ def add_logout_button(self, parent):
     def add_language_menu(self):
```

Comment:
The test in this PR is still valid and the language link should still use the object_preview_url should be used for consistency. The following PR enables the features in Versioning: https://github.com/divio/djangocms-versioning/pull/223